website/prereg/utils.py

```diff
@@ -1,9 +1,38 @@
 from modularodm import Q
 
-def get_prereg_schema():
+PREREG_CAMPAIGNS = {
+    'prereg': 'Prereg Challenge',
+    'erpc': 'Election Research Preacceptance Challenge',
+}
+
+PREREG_LANGUAGE = {
+    'prereg': {
+        'description': 'The process of preregistering your plans is beneficial to both the scientific field and to you, the scientist. By writing out detailed data collection methods, analysis plans, and rules for excluding or missing data, you can make important decisions that affect your workflow earlier, without the biases that occur once the data are in front of you.',
```

Comment:
This is probably fine, but seems like it's reinventing the wheel a bit. It might be more consistent with the existing code base to use a base template, e.g. `prereg_landing_base.mako`, that includes hooks to modify this content.

```mako
<%inherit file="prereg_landing_base.mako" />

<%def name="description()">
    Lorem ipsum dolor sit amet, <b>consetetur<b> sadipscing elitr    
</%def>
```

This will make it easier to include HTML in these fields.